Doc/tutorial/controlflow.rst

```diff
@@ -66,20 +66,20 @@ they appear in the sequence.  For example (no pun intended):
    window 6
    defenestrate 12
 
-If you need to modify the sequence you are iterating over while inside the loop
-(for example to duplicate selected items), it is recommended that you first
-make a copy.  Iterating over a sequence does not implicitly make a copy.  The
-slice notation makes this especially convenient::
-
-   >>> for w in words[:]:  # Loop over a slice copy of the entire list.
-   ...     if len(w) > 6:
-   ...         words.insert(0, w)
-   ...
-   >>> words
-   ['defenestrate', 'cat', 'window', 'defenestrate']
-
-With ``for w in words:``, the example would attempt to create an infinite list,
-inserting ``defenestrate`` over and over again.
+Code that modifies a collection while iterating over that same collection can
+be tricky to get right.  Instead, it is usually more straight-forward to loop
+over a copy of the collection or to create a new collection::
```

Comment:
The three lines are clearer and more correct in giving both strategies.

Reply:
You're happy with the proposed wording, correct?

Reply:
Yes.  There is no checkbox for "(*) Approve either as is as a definite and acceptable improvement over the status quo or after possible further improvements based on my comments -- you decide."

In this case, you considered each suggestion and and gave a short coherent reason why not, one which I can consider when writing or reviewing other doc patches.

I appreciate you tackling some of these lingering doc issues which have seemingly been blocked by the quest for perfection even when improvement is clearly possible.